src/lib/nodemailer.ts

```diff
@@ -20,6 +10,16 @@ export const sendPassRecoveryMail = async ({recipient, token}: any) => {
       <p>Password recovery session will be closed in 5 minutes.</p>
   `
 
+  const transporter = nodemailer.createTransport({
```

Comment:
Is the app still functioning when you move all these lines to the top?

Reply:
yes